pyproj/_crs.pyx

```diff
@@ -489,6 +490,34 @@ cdef class Ellipsoid(Base):
         """
         return Ellipsoid.from_authority("EPSG", code)
 
+    @staticmethod
+    def from_string(proj_string):
+        """
+        Create an Ellipsoid from a projection string.
```

Comment:
I find it a bit unclear what a "projection string" is, because I would intuitively think about a "proj string", so something like "+ellps=WGS84", which I suppose is not actually supported?

Reply:
I guess I should say a valid ellipsoid projection string. As far as I could tell, it takes a wkt and the urn, but not sure about what else it accepts.

Reply:
I agree examples here are warranted.

Reply:
I would also not use the word *projection* in "projection string", because a) an ellipsoid is not a projection and b) it does let me think of a *proj* string, which is not accepted.

Reply:
How about an `ellipsoid string`?

Reply:
I think I resolved it. Let me know if I didn't. Thanks for the comment!

Reply:
> How about an ellipsoid string?

Sounds good! And certainly the examples help a lot!